API/Controllers/RoleController.cs

```diff
@@ -39,7 +42,7 @@ public RoleController(IRoleService roleService, IUserService userService, IMappe
         ///     Get all Roles.
         /// </summary>
         /// <returns></returns>
```

Comment:
Empty returns tag (while this method does return something).
Please, also fix the other empty returns tags in this file :)